projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-totals.component.migration.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ACTIVE_CART_SERVICE,
+  ANGULAR_ROUTER,
+  CART_TOTALS_COMPONENT,
+  ROUTER,
+  SPARTACUS_CORE,
+  SPARTACUS_STOREFRONTLIB,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CART_TOTALS_COMPONENT_MIGRATION: ConstructorDeprecation = {
+  // projects/storefrontlib/cms-components/cart/cart-totals/cart-totals.component.ts
+  class: CART_TOTALS_COMPONENT,
+  importPath: SPARTACUS_STOREFRONTLIB,
+  deprecatedParams: [
```

Comment:
I believe for deprecatedParams, you need to state the entire constructor, and in add/removeParams, you add or remove the ones you want.

Therefore, add another object for Router
```suggestion
  deprecatedParams: [ {
      className: ACTIVE_CART_SERVICE,
      importPath: SPARTACUS_CORE,
    },  {
      className: ROUTER,
      importPath: ANGULAR_ROUTER,
    }] 
```